ts/hooks/use_web3.ts

```diff
@@ -0,0 +1,68 @@
+import { useWeb3React } from '@web3-react/core';
+import { useEffect, useState } from 'react';
+
+import { injected } from 'ts/utils/connectors';
+
+export function useEdgerConnect(): boolean {
```

Comment:
nit: `useEdgerConnect` -> `useEagerConnect`

Reply:
I don't understand this

Reply:
useEdgerConnect is mispelled -- should be useE**a**gerConnect

Reply:
Okay, thanks